src/db_models.py

```diff
@@ -71,7 +71,7 @@ class Event(Base):
 class Timer(Base):
     __tablename__ = 'timers'
 
-    id = Column(Integer, primary_key=True)
+    id = Column(Integer, primary_key=True, server_default=text("nextval('timer_id_seq'::regclass)"))
```

Comment:
Are you sure this is needed?

Searching says this happens automatically if primary_key=True.
I tried it as part of a different feature where this had to go, and seems to work fine.

Ideally - revert this change.
If not - let's talk about why this is needed.